### PR TITLE
LRDOCS-7549 - Site Navigation (2nd pass)

### PR DESCRIPTION
**Pull request by justinchoi001**

### CODE REVIEW

File: docs/dxp/7.x/en/site-building/site-navigation/managing-page-hierarchies.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+## Creating a Page
+
+New pages are created on the _Site Builder_ &rarr; _Pages_ page in _Site Administration_. Pages can be created as _Public Pages_ which anyone can view, or _Private Pages_ which can only be viewed by Site Members. See [Adding a Page to a Site](../creating-pages/adding-pages/adding-a-page-to-a-site.md) to learn how to add a page.
+
+Now that the page is created, it appears in the hierarchy, and you can move or organize its position there.
+
+## Organizing Pages
+
+Drag and drop pages to reorder their position in the page hierarchy (and subsequently the default navigation that users see) or to nest them as sub-pages. The page at the top of the list is the first page that users see automatically when visiting your Site. You can treat this as the site's landing page.
```
Comment (JamesAGarcia) [inline docs/dxp/7.x/en/site-building/site-navigation/managing-page-hierarchies.md:13]: To cover [LRDOCS-8563](https://issues.liferay.com/browse/LRDOCS-8563) ([LPS-114527](https://issues.liferay.com/browse/LPS-114527)), all that needs to be said is 
"You can drag and drop _**one or more**_ Pages to reorder their position in the Page hierarchy...etc." (i.e., the bolded text)
I was wondering, would you be willing to make this addition and update the the PR? Or should I open a PR to this branch with the edit?

Comment (justinchoi001) [reply docs/dxp/7.x/en/site-building/site-navigation/managing-page-hierarchies.md:13]: I've adapted this paragraph.

**Comment by sez11a** [comment]
Superseded by https://github.com/jrhoun/liferay-learn/pull/759

File: docs/dxp/7.x/en/site-building/site-navigation/navigation-menu-widget-reference.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+This section configures which pages at what level from the menu are displayed in the widget. You can choose the starting level, how many levels deep to display, and how to display sub-levels.
+
+![Select the Menu Items to Show.](./navigation-menu-widget-reference/images/04.png)
+
+| Field | Description |
+| --- | --- |
+| **Start with Menu Items In** | Select the type of Level: choose from _Level_, _Level Relative to the Current Menu Item_, or _Select Parent_.  |
+| **Number of Levels** | Select the level number to display, from _0_ to _4_. Level 0 represents the parent page, Level 1 represents the first level child page, and so forth. |
+| **Levels to Display:** | Select the number of levels to display in the navigation, from _1_ down to _Unlimited_. |
+| **Expand Sublevels:** | Select _Auto_ or _All_ from this dropdown menu; this determines if hovering the mouse over the navigation reveals navigation levels one at a time automatically or reveal all the levels at once. |
```
Comment (victorg1991) [inline docs/dxp/7.x/en/site-building/site-navigation/navigation-menu-widget-reference.md:52]: Minor nit here: Expand sublevels configuration is not used in any of the out of the box site navigation display templates. So maybe we have to add a note here telling about it, and also mentioning that custom templates could use it

**Pull request closed without merging**